### PR TITLE
Introduce 'to_numpy' / 'from_numpy' methods

### PR DESCRIPTION
**Pull request by sdrave**
This implements the proposal in #252.

### CODE REVIEW

File: src/pymor/playground/operators/numpy.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def apply(self, U, mu=None):
 
         if self.vector:
             V = super().apply(U, mu=mu)
-            return self.range.from_data(V.data)
+            return self.range.from_numpy(V.to_numpy)
```
Comment (pmli) [inline src/pymor/playground/operators/numpy.py]: Is `()` missing here?

**Comment by sdrave** [comment]
@renemilk, we have again unsuccessful checks due to decreasing coverage :(

**Comment by renemilk** [comment]
@sdrave look who's back and again not using the same config as in that other PR

**Comment by sdrave** [comment]
@pmli, I would suggest that you merge sys-mor with master soonish s.t. these interface changes propagate to your branch ..

**Pull request merged**